### PR TITLE
Adding HTTP (WIP)

### PR DESCRIPTION
**Pull request by aliceadelef**
Current state:
- coppied http_json to htto_proto, updated the collector to use staging, and the url to be the http url.
- looking into creating proto files https://developers.google.com/protocol-buffers/docs/reference/ruby-generated
- not much else 🤷‍♀️

### CODE REVIEW

File: example.rb [hunk 1]
```diff
@@ -4,9 +4,9 @@
 SimpleCov.start
 require 'lightstep'
 
-access_token = '{your_access_token}'
+access_token = '61c90a839a46c996e79c56afa1f116b8'
```
Comment (splittingred) [inline example.rb]: I'm guessing this was unintentional?

File: lightstep.gemspec [hunk 2]
```diff
@@ -17,6 +17,10 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   spec.add_dependency 'concurrent-ruby', '~> 1.0'
+  spec.add_dependency 'google-protobuf', '~> 3.1'
+  spec.add_dependency 'google-protoc', '~> 3.0.0.alpha.6.0.0'
+  spec.add_dependency 'google-api-client', '~> 0.24.3'
+  spec.add_dependency 'googleapis-common-protos', '~> 1.3'
```
Comment (splittingred) [inline lightstep.gemspec:23]: Hard-tagging these at the patch-level (or minor, for that matter) here would prevent any downstream libraries/systems from using newer versions, right? Also, do you need the `protoc` gem as a runtime dep?

File: example.rb (hunk 1 above)
Comment (aliceadelef) [reply example.rb]: yup, thanks for catching.

File: lightstep.gemspec (hunk 2 above)
Comment (aliceadelef) [reply lightstep.gemspec:23]: Hi thanks for looking, and no probably not. I think I need to close this PR, it is a half finished work in progress. Apologies for leaving it up.

**Pull request closed without merging**